File: app/src/main/java/com/leoart/koreanphrasebook/ui/chapters/models/Phrase.java
package com.leoart.koreanphrasebook.ui.chapters.models;

/**
 * Created by bogdan on 11/5/16.
 */
public class Phrase {
    private String word;
    private String translation;
    private String transcription;

    public Phrase(String word, String translation, String transcription) {
        this.word = word;
        this.translation = translation;
        this.transcription = transcription;
    }

    public String getWord() {
        return word;
    }

    public String getTranslation() {
        return translation;
    }

    public String getTranscription() {
        return transcription;
    }
}

File: app/src/main/java/com/leoart/koreanphrasebook/ui/chapters/models/Phrase.kt
package com.leoart.koreanphrasebook.ui.chapters.models

import com.google.firebase.database.Exclude
import java.util.*

/**
 * Created by bogdan on 11/5/16.
 */
class Phrase(val word: String, val translation: String, val transcription: String, val index: Int) {

    constructor() : this("", "", "", 0)

    @Exclude
    fun toMap(): HashMap<String, Any> {
        val map = HashMap<String, Any>()
        map.put("word", word)
        map.put("translation", translation)
        map.put("transcription", transcription)
        map.put("index", index)
        return map
    }
}
